docs/docs/css.md

```diff
@@ -23,7 +23,7 @@ Here is a CSS example that modifies some of the CSS Variables.
 }
 ```
 
-You can find all CSS Variables using the Web Inspector or navigate to https://github.com/coralproject/talk/blob/main/CSS_VARIABLES.md for a comprehensive list and additional information.
+You can find all CSS Variables using the Web Inspector or navigate to [https://github.com/coralproject/talk/blob/main/CSS_VARIABLES.md](https://github.com/coralproject/talk/blob/main/client/CSS_VARIABLES.md) for a comprehensive list and additional information.
```

Comment:
Redirecting from `https://github.com/coralproject/talk/blob/main/CSS_VARIABLES.md` to `https://github.com/coralproject/talk/blob/main/client/CSS_VARIABLES.md` might be confusing, would you mind making them match?

i.e.
```
[https://github.com/coralproject/talk/blob/main/client/CSS_VARIABLES.md](https://github.com/coralproject/talk/blob/main/client/CSS_VARIABLES.md)
```

Reply:
```suggestion
You can find all CSS Variables using the Web Inspector or navigate to [https://github.com/coralproject/talk/blob/main/client/CSS_VARIABLES.md](https://github.com/coralproject/talk/blob/main/client/CSS_VARIABLES.md) for a comprehensive list and additional information.
```